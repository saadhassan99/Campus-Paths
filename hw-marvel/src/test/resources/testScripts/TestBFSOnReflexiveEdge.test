# test BFS on ReflexiveEdge data to make sure MarvelPath 
# doesn't allow building reflexive edges

# build the graph
LoadGraph ReflexiveEdge ReflexiveEdge.tsv

# test BFS order

# find the shortest path A to C
FindPath ReflexiveEdge A C

# find the shortest path C to A
FindPath ReflexiveEdge C A

# find the shortest path A to A
FindPath ReflexiveEdge A A