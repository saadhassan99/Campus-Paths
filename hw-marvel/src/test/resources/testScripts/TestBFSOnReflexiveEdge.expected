# test BFS on ReflexiveEdge data to make sure MarvelPath 
# doesn't allow building reflexive edges

# build the graph
loaded graph ReflexiveEdge

# test BFS order

# find the shortest path A to C
path from A to C:
A to B via AB
B to C via BC

# find the shortest path C to A
path from C to A:
C to B via BC
B to A via AB

# find the shortest path A to A
path from A to A:
